ui/app/pages/asset/components/native-asset.js

```diff
@@ -1,19 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-
 import TransactionList from '../../../components/app/transaction-list';
 import { EthOverview } from '../../../components/app/wallet-overview';
-import { getSelectedIdentity } from '../../../selectors/selectors';
+import {
+  getSelectedIdentity,
+  getCurrentChainId,
+  getRpcPrefsForCurrentProvider,
+} from '../../../selectors/selectors';
+import { showModal } from '../../../store/actions';
+import getAccountLink from '../../../../lib/account-link';
 import { DEFAULT_ROUTE } from '../../../helpers/constants/routes';
-
 import AssetNavigation from './asset-navigation';
+import TokenOptions from './token-options';
```

Comment:
This might be a good time to rename this to something other than "token options", since it's now being used for native assets as well.

Reply:
That makes sense to me. I'll make this change!

Reply:
@Gudahtt so I may have gone beyond what you were thinking with renaming classnames and such. There is a issue with the verify-locales process no longer seeing tokenOptions present in the UI, but I'm not sure how or if the translations should change in the locales files if we do wish to change these classnames to match the new more generic verbiage?

Reply:
Looks like E2E tests are failing from these classname changes, thought I'm not totally clear why yet

Reply:
It looks like the E2E errors are the same as the locales issues. @brad-decker @Gudahtt Should I fully revert the name changes because of the translation blocker for now? Or just revert the classnames?